lib/routes.php

```diff
@@ -0,0 +1,19 @@
+<?php
+return [
+    'routes' => [
```

Comment:
je hebt een nieuwe routes.php toegevoegd in je pr onder lib/routes.php maar deze (GET /events/subscriptions) kan ik niet bevragen, krijg ik een 404. Hoe is deze lib/routes.php anders dan de file appinfo/routes.php?